labeled_data/bot/index.yml

```diff
@@ -0,0 +1,12 @@
+name: GitHub Bot
+type: Bot
+label:
```

Comment:
This should be `data` but not `label`, please check the existing data: https://github.com/X-lab2017/open-digger/blob/master/labeled_data/communities/xlab/index.yml